app/widget/timebased/timebasedwidget.h

```diff
@@ -54,6 +54,28 @@ class TimeBasedWidget : public TimelineScaledWidget
 
   virtual bool eventFilter(QObject* object, QEvent* event) override;
 
+  // Temp file location
```

Comment:
@itsmattkc Do you think it would make sense to move this and the marker add class to their own `markerundo.h` file alongside `marker.h`?